test/Chainweb/Test/MultiNode.hs

```diff
@@ -523,8 +673,13 @@ consensusStateSummary s
     avg :: Foldable f => Real a => f a -> Double
     avg f = realToFrac (sum $ toList f) / realToFrac (length f)
 
-    median :: Foldable f => Ord a => f a -> a
-    median f = (!! ((length f + 1) `div` 2)) $ L.sort (toList f)
+    median :: (Ord a, Integral a) => (Foldable f) => f a -> a
+    median f = case L.sort (toList f) of
+      [] -> error "median: empty list"
+      xs ->
+        if length xs `mod` 2 == 1
+        then xs !! (length f `div` 2)
+        else ((xs !! (length xs `div` 2 - 1)) + (xs !! (length xs `div` 2))) `div` 2
```

Comment:
I would honestly fold this into a guard:

```
xs | length xs `mod` 2 == 1 -> ...
   | otherwise -> ....
```

and save a line, but that's just a readability nit, so feel free to abandon.